build.rs

```diff
@@ -131,8 +131,9 @@ fn platform() -> &'static str {
 
 fn download_ninja_gn_binaries() {
   let root = env::current_dir().unwrap();
-  // target/debug//build/rusty_v8-d9e5a424d4f96994/out/
-  let out_dir = env::var_os("OUT_DIR").unwrap();
+  let out_dir = env::var_os("OUT_DIR").expect(
+    "OUT_DIR expected target/debug//build/rusty_v8-xxxxxxxxxxxxxxxx/out/.",
```

Comment:
I don't want to be nitpicky but this is just as cryptic as the original error message.
Maybe...
`The 'OUT_DIR' environment is not set (it should be something like 'target/debug/rusty_v8-{hash}')`.

Reply:
Thank you so much! 
Corrected.